source/lac/sparsity_tools.cc

```diff
@@ -1040,42 +1032,51 @@ namespace SparsityTools
     IndexSet owned(start_index.back());
     owned.add_range(start_index[myid], start_index[myid] + rows_per_cpu[myid]);
 
+    distribute_sparsity_pattern(dsp, owned, mpi_comm, myrange);
+  }
+
+
+
+  void
+  distribute_sparsity_pattern(DynamicSparsityPattern &dsp,
+                              const IndexSet &        owned,
+                              const MPI_Comm &        mpi_comm,
+                              const IndexSet &        myrange)
```

Comment:
Would you mind changing these variable names to match the ones in the declaration? It makes the code a bit easier to read.

Reply:
Definitely, I will do it now!

Reply:
Done!